nexus/src/app/mod.rs

```diff
@@ -245,6 +280,55 @@ impl Nexus {
         }
     }
 
+    // Called to hand off management of external servers to Nexus.
+    pub(crate) async fn set_servers(
+        &self,
+        external_servers: ExternalServers,
+        internal_server: DropshotServer,
```

Comment:
Server lifetime question - will the servers passed into this method already be bound to their listening ports, or does that happen later? (If already bound, does that require that they be on different ports than any existing servers, if we close those as a part of this method?)

Reply:
The servers passed into this method definitely *can* be bound to listening ports - but if they were, that failure would happen at bind time.

The `set_servers` method is being called during Nexus initialization, so this "close" is just defensive coding, not really a necessary aspect of setup.